saleor/graphql/payment/mutations.py

```diff
@@ -141,6 +169,36 @@ def mutate(cls, root, info, payment_id, amount=None):
         return PaymentCapture(payment=payment, errors=errors)
 
 
+class PaymentCharge(BaseMutation):
+    payment = graphene.Field(Payment, description='Updated payment')
+
+    class Arguments:
+        payment_id = graphene.ID(required=True, description='Payment ID')
+        payment_token = graphene.String(
```

Comment:
Shouldn't this field be required? The same in `PaymentAuthorize`.

Reply:
Good catch, thanks!